vhdl_lang/src/analysis/named_entity/arena.rs

```diff
@@ -120,6 +121,10 @@ impl<'a> FinalArena {
     pub fn clear(&mut self) {
         self.refs.clear();
     }
+
+    pub fn items(&self) -> impl Iterator<Item = &Arc<LocalArena>> {
```

Comment:
Do we really need to iterate over LocalArenas? I would prefer if it was private. Maybe add a function to iterate over all entities?

Reply:
Turns out this was superfluous as well :D I removed the function altogether